config/updates.jenkins.io.yaml

```
@@ -11,99 +9,71 @@ mirrorbits-lite:
       - host: azure.updates.jenkins.io
         paths:
           - path: /
-            serviceNameSuffix: files
+            backendService: httpd
           - path: /.*[.](deb|hpi|war|rpm|msi|pkg|sha256|md5sum|zip|gz|pdf|json|svg|sh|jpeg|ico|png|html)$
             pathType: ImplementationSpecific
+            backendService: mirrorbits
     tls:
       - secretName: updates-jenkins-io-tls
         hosts:
           - azure.updates.jenkins.io
 
+  storage:
+    enabled: true
+    claimNameTpl: '{{ include "mirrorbits-parent.pvc-name" $ }}'
+    storageClassName: azurefile-csi-premium
+    storageSize: 2Gi # See file share size in https://github.com/jenkins-infra/azure/blob/main/updates.jenkins.io.tf
```

```suggestion
    storageSize: 2Gi  # See file share size in https://github.com/jenkins-infra/azure/blob/main/updates.jenkins.io.tf
```
Fix "too few spaces before comment" yamllint warning